### PR TITLE
26 navbar

### PR DESCRIPTION
**Pull request by TimW90**
Added navbar

### CODE REVIEW

File: frontend/src/components/Navbar.tsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+            <div className="form-control text-align: right">
+                <input
+                    type="text"
+                    placeholder="Search"
+                    className="input input-bordered w-24 md:w-auto"
+                />
+            </div>
+        </div>
+        <div className="flex-none">
+            <div className="dropdown dropdown-end pr-1">
```
Comment (melleg) [inline frontend/src/components/Navbar.tsx:14]: Margin instead of padding please

File: frontend/src/components/Navbar.tsx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+                <div
+                    tabIndex={0}
```
Comment (melleg) [inline frontend/src/components/Navbar.tsx:16]: Tabindex unnecessary here

File: frontend/src/components/Navbar.tsx [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+                                type="button"
+                                className="btn btn-primary btn-block"
+                            >
+                                View cart
+                            </button>
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <div className="dropdown dropdown-end">
```
Comment (melleg) [inline frontend/src/components/Navbar.tsx:56]: Remove profile picture dropdown

File: frontend/src/App.tsx [hunk 4]
```diff
@@ -1,7 +1,13 @@
-import Product from './components/Product';
+import Navbar from './components/Navbar';
+import Products from './components/Products';
 
 const App = () => {
-    return <Product />;
+    return (
+        <div>
```
Comment (melleg) [inline frontend/src/App.tsx:6]: Replace with react fragment (`<></> instead of <div></div>`)

**Review by bartspreij — APPROVED** [review]

**Review by melleg — APPROVED** [review]

**Pull request merged**